website/docs/docusaurus.config.js

```diff
@@ -96,5 +96,78 @@ module.exports = {
   ],
   stylesheets: [
     'https://use.typekit.net/gyc5wys.css'
+  ],
+  plugins: [
+    // [
+    //   'docusaurus-plugin-typedoc',
```

Comment:
Are we leaving these comments as reminders somehow?

Reply:
Oh, yes, I should have been more clear about that. The current version does not support multiple instances of this plugin. Eventually I think we expect to use typedoc for these other ones, but we need to upstream support first. I had set them up so I left them in there as a reminder.

Reply:
Ok got it!